game-core/src/test/java/games/strategy/triplea/xml/TestMapGameData.java

```diff
@@ -38,7 +38,9 @@
 
   GAME_EXAMPLE("GameExample.xml"),
 
-  TWW("Total_World_War_Dec1941.xml");
+  TWW("Total_World_War_Dec1941.xml"),
+
+  MINIMAP("minimap.xml");
```

Comment:
side-note, I'm thinking to propose a convention that we end enums with a trailing comma and semi on the next line. This way adding elements adds just a new line. 

In this case this would be:
```
MINIMAP("minimap.xml"),
;
```

Another side-comment, the tests that require full map data are not necessarily ideal. They involve lots of moving pieces, it's not a proper unit test if we use them. There are some additional drawbacks. Long story short, we want to in the short term see if we consolidate how many test XMLs are in use, use as few as possible, eventually get to a place where game rules can be tested ad-infinitum without an actual XML